golang/go.mod

```diff
@@ -4,13 +4,11 @@ go 1.13
 
 require (
 	github.com/golang/protobuf v1.4.2
-	github.com/google/uuid v1.1.2
-	github.com/hashicorp/go-retryablehttp v0.6.7
 	github.com/palantir/stacktrace v0.0.0-20161112013806-78658fd2d177
-	github.com/powerman/rpc-codec v1.2.2
 	github.com/sirupsen/logrus v1.6.0
 	github.com/stretchr/testify v1.6.1
-	golang.org/x/crypto v0.0.0-20201221181555-eec23a3978ad
+	golang.org/x/net v0.0.0-20190404232315-eb5bcb51f2a3 // indirect
```

Comment:
@mieubrisse I just noticed these changes - any idea what happened here?

Reply:
Yep, not a cause for concern - basically, we used to use the `crypto` library in Libs, but I pushed the logic inside Core so now it's deleting the deps